Dockerfile

```diff
@@ -13,21 +13,19 @@ RUN yum -y install epel-release \
     && yum -y install ansible sudo \
     && ansible-galaxy install -p /opt/setup/roles -r requirements.yml
 
-RUN ansible-playbook playbook.yml
+ARG OMERO_WEB_VERSION=5.6.dev7
+RUN ansible-playbook playbook.yml \
+    -e omero_web_release=$OMERO_WEB_VERSION
 
 RUN curl -L -o /usr/local/bin/dumb-init \
-    https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 && \
+    https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && \
     chmod +x /usr/local/bin/dumb-init
 ADD entrypoint.sh /usr/local/bin/
 ADD 50-config.py 60-default-web-config.sh 98-cleanprevious.sh 99-run.sh /startup/
 ADD ice.config /opt/omero/web/OMERO.web/etc/
 
-# Workaround for replace_omero
-RUN rm /opt/omero/web/OMERO.web/bin/omero && ln -s /opt/omero/web/venv3/bin/omero /opt/omero/web/OMERO.web/bin/omero
-
 USER omero-web
 EXPOSE 4080
 VOLUME ["/opt/omero/web/OMERO.web/var"]
 
-ENV OMERODIR=/opt/omero/web/OMERO.web/
```

Comment:
Not sure about this. It's set in the wrapper script. Delete it, keep it, or keep it only in the `devtest` image?

Reply:
For uses without the wrapper (i.e. with `venv3/bin/omero`, I'd keep it.) No strong feeling over here or only in devtest if the latter works, but at least in omero-server there were various requirements to have it defined earlier (e.g. pre-`omego`)